src/containers/pages/PlanAssignment/index.tsx

```diff
@@ -245,7 +259,7 @@ const PlanAssignment = (props: PlanAssignmentProps) => {
 
   return (
     <>
-      <ConnectedAssignmentMapWrapper {...AssignmentWraperProps} />
+      {isMapDisabled(plan) ? '' : <ConnectedAssignmentMapWrapper {...AssignmentWraperProps} />}
```

Comment:
Should we maybe change this to !isMapDisabled && <ConnectedAssignmentMapWrapper {...AssignmentWraperProps} />? This is so that we avoid the weird empty string output